sdk/computefleet/Azure.ResourceManager.ComputeFleet/assets.json

```diff
@@ -0,0 +1,6 @@
+{
+  "AssetsRepo": "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath": "net",
+  "TagPrefix": "net/computefleet/Azure.ResourceManager.ComputeFleet",
+  "Tag": ""
```

Comment:
Hi @ArthurMa1978, Do I need a tag here as well for ComputeFleet?

Reply:
Follow this guide https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md to add tests, once push test record this tag will be updated.